snapshot-labs/sx-monorepo
# feat: add support for offchain proposal edition

### Summary



This PR add support for edition of offchain proposals

### How to test

1. Go to an offchain pending proposal
2. Navigate to the proposal edition form
3. The form should be pre-populated with the correct data (including voting type and choices)
4. Try to edit proposals setting
5. Submit
6. It should have updated the proposal with the new data

### To-Do

- [x] Merge https://github.com/snapshot-labs/sx-monorepo/pull/73 first
